letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -912,6 +912,34 @@ OldVenvExists() {
     [ -n "$OLD_VENV_PATH" -a -f "$OLD_VENV_PATH/bin/letsencrypt" ]
 }
 
+# Given python path, version 1 and version 2, check if version 1 is outdated compared to version 2.
+# Unofficial versions (eg. 0.28.0.dev0) will be treated specifically by printing "UNOFFICIAL".
+# Otherwise, print "OUTDATED" if version 1 is outdated, or "UP_TO_DATE" if not.
+# This function relies only on installed python environment (2.x or 3.x) by certbot-auto.
+CompareVersions() {
+    "$1" - "$2" "$3" << "UNLIKELY_EOF"
+import sys
+from distutils.version import StrictVersion
+
+try:
+    current = StrictVersion(sys.argv[1])
+except:
+    sys.stdout.write('UNOFFICIAL')
+    sys.exit()
+
+try:
+    target = StrictVersion(sys.argv[2])
```

Comment:
"new" or "remote" might be a clearer name here. The latter is consistent with the name we give it at the callsite (`$REMOTE_VERSION`).

Reply:
Yes `remote` is a good word.